src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -190,4 +194,16 @@ public double getTargetQ() {
         }
         return super.getTargetQ();
     }
+
+    @Override
+    public List<Integer> getNodes() {
+        if (topologyKind == TopologyKind.NODE_BREAKER) {
+            Map<String, Set<Integer>> nodesByBus = Networks.getNodesByBus(getBus().getVoltageLevel());
+            if (nodesByBus.containsKey(getBus().getId())) {
+                nodes = nodesByBus.get(getBus().getId()).stream().toList();
+            }
+
+        }
+        return nodes;
+    }
 }
```

Comment:
Can you add a test for this method, that verifies that the result is as expected ?